cluster/cluster.go

```diff
@@ -179,7 +212,19 @@ func (c *Cluster) Create(stackBody string, s3URI string) error {
 	cfSvc := cloudformation.New(c.session)
 	s3Svc := s3.New(c.session)
 
-	return c.stackProvisioner().CreateStackAndWait(cfSvc, s3Svc, stackBody, s3URI)
+	stackTemplate, err := c.RenderTemplateAsString()
+	if err != nil {
+		return fmt.Errorf("Error while rendering template : %v", err)
+	}
+
+	uploads := map[string]string{
+		"stack.json":          stackTemplate,
+		"userdata-controller": c.UserDataController,
+		"userdata-worker":     c.UserDataWorker,
+		"userdata-etcd":       c.UserDataEtcd,
```

Comment:
etcd userdata can't be  moved to s3, have to stay in CF template to be excluded from updates